bin/test-backend.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+NAMESPACE=staging
+IMAGE_TAG=healthcheck2
```

Comment:
Use commit has for this

Reply:
The reason I have used `healthcheck2` is because this is the only image that would currently work until the PR containing endpoint for the health check gets merged.

Reply:
Since we don't actually make deployments on tests yet I will still prefer it remaining a commit hash.

Reply:
You can make these changes on the next PR though.

Reply:
ok